sdk/src/main/java/com/horizen/proposition/PublicKey25519Proposition.java

```diff
@@ -25,12 +25,24 @@
 
     public PublicKey25519Proposition(byte[] pubKeyBytes)
     {
-        if(pubKeyBytes.length != KEY_LENGTH)
+        if (pubKeyBytes.length != KEY_LENGTH)
```

Comment:
let's just call other constructor here to avoid code duplication: `PublicKey25519Proposition(pubKyBytes, false)`, same in other places.